ftw/news/browser/news_listing.py

```diff
@@ -0,0 +1,175 @@
+from Acquisition import aq_parent, aq_inner
+from DateTime import DateTime
+from DateTime.interfaces import SyntaxError as dtSytaxError
```

Comment:
I personally don't like `as`-imports that much.
I would just import the complete package:

```python
import DateTime

try:
    DateTime.DateTime(value)
except DateTime.interfaces.SyntaxError:
    raise
```

It's up to you though :wink: